src/Symfony/Component/Notifier/Bridge/Slack/Block/SlackPlainTextInputBlock.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Symfony\Component\Notifier\Bridge\Slack\Block;
+
+
+final class SlackPlainTextInputBlock extends AbstractSlackBlock
+{
+    private const LABEL_LIMIT = 150;
+    private const ACTION_ID_LIMIT = 255;
+    private const PLACEHOLDER_LIMIT = 150;
+
+    public function __construct(string $labelText, string $actionId, ?string $placeholderText = null)
+    {
+        if (\strlen($labelText) > self::LABEL_LIMIT) {
+            throw new \LengthException(\sprintf('Maximum length for the label text is %d characters.', self::LABEL_LIMIT));
```

Comment:
Do we have length exception in the slack bridge or in the notifier bridge?

Reply:
I'm not sure I understand the question.

I did it like this:

https://github.com/symfony/symfony/blob/195f1f123d1328ef534f70e4605948049120ae73/src/Symfony/Component/Notifier/Bridge/Slack/Block/SlackHeaderBlock.php#L27

The documentation for the input on Slack:

https://api.slack.com/reference/block-kit/block-elements#input"

Reply:
All good, thanks